### PR TITLE
Implement the same thing without additional viewmodel

### PR DESCRIPTION
**Pull request by kornelijepetak**
@juanlaube this is what @willsb wanted.

You can review it, approve it and squash it into your branch if you want.
That's an insta-approval from @willsb I think, since he said:
`And yes, that’s how I’d implement it :)`

### CODE REVIEW

File: Toggl.Giskard/Views/MainRecyclerView.cs [hunk 1]
```diff
@@ -26,11 +26,11 @@ public TimeEntriesLogViewModel TimeEntriesLogViewModel
             get => MainRecyclerAdapter.TimeEntriesLogViewModel;
             set => MainRecyclerAdapter.TimeEntriesLogViewModel = value;
         }
-
-        public MainLogFooterViewModel MainLogFooterViewModel
+
```
Comment (simonrozsival) [inline Toggl.Giskard/Views/MainRecyclerView.cs]: 🔥

Comment (kornelijepetak) [reply Toggl.Giskard/Views/MainRecyclerView.cs]: done. 👍

File: Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs [hunk 2]
```diff
@@ -240,7 +238,7 @@ private void setRunningEntry(IDatabaseTimeEntry timeEntry)
             StartTimeEntryCommand.RaiseCanExecuteChanged();
             EditTimeEntryCommand.RaiseCanExecuteChanged();
 
-            MainLogFooterViewModel.IsRunning = CurrentTimeEntryId.HasValue;
+            RaisePropertyChanged(nameof(IsTimeEntryRunning));
```
Comment (simonrozsival) [inline Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: Wouldn't it be possible to add just `[DependsOn(...)]` to the property?

Comment (kornelijepetak) [reply Toggl.Foundation.MvvmCross/ViewModels/MainViewModel.cs]: I am not sure. At first I've tried it like that, but it didn't work for some reason, and I am not sure whether Fody will touch private setters: `CurrentTimeEntryId` has one.

**Pull request merged**